Update faq slack info (#528)
kokkos/kokkos-core-wiki: docs/source/faq.rst

FAQ
###

**How do I join the Kokkos slack channel?**

>   You can find the slack channel at `kokkosteam.slack.com <https://kokkosteam.slack.com>`_. Register a new account with your email. We reached the limit of whitelisted organizations, but every member of the Slack can invite more people. If no one you know is in the Slack you can contact the Kokkos maintainers (their emails are in the LICENSE file).

Reviewer: ```suggestion
  You can find the slack channel at `kokkosteam.slack.com <https://kokkosteam.slack.com>`_. Register a new account with your email. We reached the limit of whitelisted organizations, but every member of the Kokkos Slack workspace can invite more people. If no one you know is in the Slack workspace you can contact the Kokkos maintainers (their emails are in the LICENSE file).
```

>   You can find the slack channel at `kokkosteam.slack.com <https://kokkosteam.slack.com>`_. Register a new account with your email. We reached the limit of whitelisted organizations, but every member of the Slack can invite more people. If no one you knows is in the Slack you can contact the Kokkos maintainers (their emails are in the LICENSE file).

Reviewer: ```suggestion
  You can find the slack channel at `kokkosteam.slack.com <https://kokkosteam.slack.com>`_. Register a new account with your email. We reached the limit of whitelisted organizations, but every member of the Slack can invite more people. If no one you know is in the Slack you can contact the Kokkos maintainers (their emails are in the LICENSE file).
```

Reviewer 2: ```suggestion
  You can find the slack channel at `kokkosteam.slack.com <https://kokkosteam.slack.com>`_. Register a new account with your email. We reached the limit of whitelisted organizations, but every member of the Slack workspace can invite more people. If no one you know is in the Slack workspace you can contact the Kokkos maintainers (their emails are in the LICENSE file).
```

Reviewer: ignoring

Reviewer 2: > [...] every member of the Slack can invite more people. 

Just doesn't sound correct.

Reviewer: Updated
